src/arrayMethodSort.js

```diff
@@ -5,7 +5,31 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    if (typeof compareFunction === 'function') {
```

Comment:
make sense to set default value for `compareFunction ` because to need to avoid code duplication